Makefile

```diff
@@ -25,10 +37,15 @@ E2E_BUILDAH_PVC_NAME ?= task-buildah
 # buildah task e2e test variables, image path, containerfile_path
 E2E_BUILDAH_CONTAINERFILE_PATH ?= /workspace/source/Dockerfile
 E2E_BUILDAH_IMAGE ?= test-buildah
-E2E_BUILDAH_REGISTRY ?= docker://registry.registry.svc.cluster.local:32222/test-buildah:latest
+E2E_BUILDAH_REGISTRY ?= registry.registry.svc.cluster.local:32222/test-buildah:latest
```

Comment:
Let's rename this to reflect is target container registry, used for `buildah` tests. Also, we can reuse the registry part and compose afterwards.

Reply:
On this case is also good to have a top comment with a complete description.

Reply:
This one was done by Anand so I did not touch it fearing any loss of functionality or malfunctionality in his task on editing it.